src/main/java/com/snowflake/kafka/connector/internal/SnowflakeTelemetryService.java

```diff
@@ -38,6 +38,14 @@
    */
   void reportKafkaFatalError(String errorDetail);
 
+  /**
+   * report connector usage
```

Comment:
If it contains all connector usage, we may need to rethink about the name of `SnowflakeTelemetryPipeStatus`. if it only related to `pipe`, then we need update this comments